charts/iits-admin-dashboard/templates/servicemonitor-uptime.yaml

```diff
@@ -0,0 +1,63 @@
+{{- with .Values.uptimeMonitor }}
+{{ $defaults := .defaults }}
+{{- if .enabled | toString | eq "true" }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ printf "uptime-admin-dashboard" | replace "_" "-" | trunc 63 | trimSuffix "-" }}
+spec:
+  endpoints:
+    - port: http
+      scheme: {{ $.Values.uptimeMonitor.scheme }}
+    {{- if $.Values.uptimeMonitor.bearerTokenFile }}
+      bearerTokenFile: {{ $.Values.uptimeMonitor.bearerTokenFile }}
+    {{- end }}
+    {{- if $.Values.uptimeMonitor.tlsConfig }}
+      tlsConfig: {{ toYaml $.Values.uptimeMonitor.tlsConfig | nindent 6 }}
+    {{- end }}
+      path: {{ $.Values.uptimeMonitor.path }}
+      interval: {{ .interval | default $defaults.interval }}
+      scrapeTimeout: {{ .scrapeTimeout | default $defaults.scrapeTimeout }}
+      honorTimestamps: {{ .honorTimestamps | default $defaults.honorTimestamps }}
+      params:
+        module:
+          - {{ .module | default $defaults.module }}
+        target:
+          - {{ $.Values.ingress.host }}
```

Comment:
this does not work, the target needs to consist of schema + host + path.
You are also not iterating over what the actual endpoints the ingress exposes, but are simply hardcoding a single (wrong) value.